website/blog/2021-05-20-GAAD-One-Year-Later.md

```diff
@@ -93,29 +61,54 @@ Disables click functionality when the component has a disabled prop
 
 ### Selected State Announcement
 
-Another multi-component issue, Selected State would not announce when focused on. Now upon focus if the AccessibilityState is set to selected or if the component is changed to selected, it will announce selected.
+There were some components that did not announce their selection when in focus. This behavior has now been fixed when the component is in focus and the AccessibilityState is set to selected or the component is changed to selected.
 
-Announces when Disabled
+Announces when Selected
 
 -   Button
 
 -   TextInput
 
 ### Accessibility Timeout Setting
 
-This issue added the ability to query AccessibilityManager.getRecommendedTimeoutMillis(). This queries the "Time to take action" before the UI elements auto-dismisses or auto-progresses. Prior to this there was no way to query this with the AccessibilityInfo API on Android.
+There was previously no way to query the accessibility timeout setting on Android. The fix added the ability to query AccessibilityManager.getRecommendedTimeoutMillis(). This queries the "Time to take action" before the UI elements auto-dismisses or auto-progresses. 
 
 Documentation Additions
 -----------------------
+The React Native documentation must be updated to reflect each addition or change to the available APIs. The new addition to the React Native documentation covered the addition of [getRecommendedTimeoutMillis()](https://reactnative.dev/docs/next/accessibilityinfo#getrecommendedtimeoutmillis-android) to AccessibilityInfo.
 
-With every new addition to React Native the documentation must be updated to reflect those additions or changes. New additions to the React Native website as part of this initiative are listed below.
+Community Involvement
+---------------------
```

Comment:
There are still some whitespaces and formatting issues in this section:

<img width="746" alt="Screenshot 2021-05-19 114626" src="https://user-images.githubusercontent.com/719641/118792367-e36bd700-b897-11eb-9638-525bb4cdc79f.png">

Also authors links/mentions style are non-unified and some of URL are incorrect, ex:
* `http://fabriziobertoglio1987`

Reply:
Thank you I will tackle those.

Reply:
Fixed! I think in tackling the linting issues I took out some spaces that should not have been removed.